### PR TITLE
Update CDSL.md

### PR DESCRIPTION
**Pull request by GarvTambi**

### CODE REVIEW

**Comment by GarvTambi** [comment]
@orensbruli Please Review

File: doc/CDSL.md [hunk 1]
```diff
@@ -1 +1,22 @@
 # CDSL
+CDSL is Component Definition Language used to create and define component in Robocomp. With CDSL you specify how the component will communicate with the world. 
+
+A basic cdsl file can be generated as given below. Go to a folder and type the following commands to make a directory and generate a component :
+https://robocomp.github.io/web/gsoc/2017/love/post5
```
Comment (orensbruli) [inline doc/CDSL.md]: robocompdsl is not currently generating code in js.
It's not a good link for command examples.

File: doc/CDSL.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+
+So after this we can compile the component using the following command :
+robocompdsl myComponent.cdsl .
+
+So this generates two new folders :
```
Comment (orensbruli) [inline doc/CDSL.md]: And many more files that should be explained here.

File: doc/CDSL.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+
+1. etc
+2. src
+
+The src folder contains a folder and two files:
+1. UI (folder)
+2. component.js
```
Comment (orensbruli) [inline doc/CDSL.md]: .js is not a good example for cdsl

File: doc/CDSL.md [hunk 4]
```diff
@@ -1 +1,22 @@
...
+3. makefile
+
+So now our basic component is ready. To make the component we sue the makefile. Just type the following commands:
+cd src
```
Comment (orensbruli) [inline doc/CDSL.md]: You don't need to go to src dir to do this.

File: doc/CDSL.md [hunk 5]
```diff
@@ -1 +1,22 @@
...
+make
```
Comment (orensbruli) [inline doc/CDSL.md]: We are using cmake and is a good practice to make the compilation on a new "build" directory to keep the component dir as clean as possible.

File: README.md [hunk 6]
```diff
@@ -54,6 +54,8 @@ It is recommendable to install the following packages::
 --> 
 Note: One of the main tools of Robocomp, robocompdsl is using pyparsing and the current code doesn't work with 2.4 version of this library. With the previous commands, we are installing the 2.2 version (python-pyparsing=2.2.0+dfsg1-2). If you have a more recent version of pyparsing installed with apt or pip we recommend you to uninstall it and install the 2.2 version. You can check your current version of pyparsing with this command:
 
+Unstall the latest version by using the following command : sudo pip2 uninstall pyparsing and again install it by using this command : sudo pip2 install pyparsing==2.2
```
Comment (orensbruli) [inline README.md]: you have put this text before the command and the way of uninstalling the package will depend on how they have previously installed so we can't assume it's with pip.

**Comment by orensbruli** [comment]
@GarvTambi  can you answer the review and comments, please?

File: doc/CDSL.md (hunk 3 above)
Comment (GarvTambi) [reply doc/CDSL.md]: Ok...I take this point in my bucket of knowledge.
Thanks!!

File: README.md (hunk 6 above)
Comment (GarvTambi) [reply README.md]: I understand !!

**Comment by GarvTambi** [comment]
@orensbruli  May I close this Pull Request.

**Pull request closed without merging**